app/views/home/_deploy.html.erb

```diff
@@ -24,9 +24,12 @@
                 <%= link_to status.short_sha, status.commit_url %>
               </span>
             </div>
-            Deployed by <em><%= status.user %></em>
-            <%= time_ago_in_words(status.timestamp) %> ago.
-            View <%= link_to 'pending changes', status.pending_url %>.
+            <%= t('home.deployed_by_html',
+              user: status.user,
```

Comment:
possible to keep the `em` in the view? Ideally, we have as little HTML as possible in the translation files